setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import find_packages, setup
+
+# Package meta-data.
+NAME = 'RePoE'
+DESCRIPTION = 'Repository of Path of Exile resources for tool developers'
+URL = 'https://github.com/brather1ng/RePoE'
+EMAIL = ''
+AUTHOR = 'brather1ng'
+REQUIRES_PYTHON = '>=3.6.0'
+VERSION = '1.0.0'
+
+# What packages are required for this module to be executed?
+REQUIRED = [
+   'PyPoe'
+]
+
+# What packages are optional?
+EXTRAS = {
+    # 'fancy feature': ['django'],
+}
+
+setup(
+    name=NAME,
+    version=VERSION,
+    description=DESCRIPTION,
+    author=AUTHOR,
+    author_email=EMAIL,
+    python_requires=REQUIRES_PYTHON,
+    url=URL,
+    py_modules=['mypackage'],
```

Comment:
This should be something proper. Maybe
```suggestion
    py_modules=find_packages(exclude=['data', 'docs']),
```
? Though I don't know much about Python packaging.

Reply:
Since docs and data have no `__init__.py` they won't be considered as modules.

Reply:
How about just `py_modules=find_packages(),` then? At least `'mypackage'` shouldn't be there.

Reply:
Yeah - good point.